server/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const bodyParser= require('body-parser');
+const app = express();
+const port = 1022;
+const Home = require('../database/index');
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost:27017/homes', {useUnifiedTopology: true, useNewUrlParser: true, useCreateIndex: true });
+
+//body parser
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+//static
+app.use(express.static('public'));
+
+//routes
+app.get('/homes', function (req, res) {
```

Comment:
You database should offer a way to view 1 home's info at a time (instead of sending all data in our database to the frontend).